pkg/blobfuse/controllerserver.go

```diff
@@ -144,28 +144,17 @@ func (d *Driver) DeleteVolume(ctx context.Context, req *csi.DeleteVolumeRequest)
 		resourceGroupName = d.cloud.ResourceGroup
 	}
 
-	accountKey, err := d.cloud.GetStorageAccesskey(accountName, resourceGroupName)
-	if err != nil {
-		return nil, fmt.Errorf("no key for storage account(%s) under resource group(%s), err %v", accountName, resourceGroupName, err)
-	}
-
 	klog.V(2).Infof("deleting container(%s) rg(%s) account(%s) volumeID(%s)", containerName, resourceGroupName, accountName, volumeID)
-	client, err := azstorage.NewBasicClientOnSovereignCloud(accountName, accountKey, d.cloud.Environment)
-	if err != nil {
-		return nil, err
-	}
-	blobClient := client.GetBlobService()
-	container := blobClient.GetContainerReference(containerName)
-	// todo: check what value to add into DeleteContainerOptions
 	err = wait.ExponentialBackoff(d.cloud.RequestBackoff(), func() (bool, error) {
-		_, err := container.DeleteIfExists(nil)
+		_, err = d.client.Delete(ctx, resourceGroupName, accountName, containerName)
```

Comment:
is it identical?

Reply:
Currently, the effect is the same. Just replace the old SDK to the new one.